roles/splunk_common/tasks/enable_service.yml

```diff
@@ -31,33 +27,39 @@
 - name: "Copy Splunkd unit file - Linux (systemd)"
   template:
     src: Splunkd.service.j2
-    dest: /etc/systemd/system/Splunkd.service
+    dest: "/etc/systemd/system/{{ splunk_service_name }}"
     owner: "{{ privileged_user }}"
     group: "{{ privileged_user }}"
     mode: 0644
   become: yes
   become_user: "{{ privileged_user }}"
   when:
     - ansible_system is match("Linux")
-    - pid1.stdout.find("systemd") != -1
+    - splunk_systemd
     - installed_splunk_version[0] is version("7.2.2", "<")
 
+- name: "Disable cgroup systemd settings in Docker"
```

Comment:
Hmm I was going to suggest this does not get included in here, but it seems like 8.0.0 has decided to deviate from the systemd unit file from 7.3.x: https://docs.splunk.com/Documentation/Splunk/8.0.0/Admin/RunSplunkassystemdservice

The docs seem to suggest that upon startup, Splunk 8.0.0 will automatically correct itself - so I'm still not sure if we should be responsible for doing it here, but it's something worth testing. There might also be some impact to workload management here as well? @bb03 
 
Also cc'ing @rcabral-splunk since this will directly impact your work as well.

Reply:
oh yeah sorry. I intended to put a comment in there that I couldn't find a way to get that to work at least in my setup. I'm still kinda new to the whole docker thing so it could just be I don't have something set up right.

Reply:
I'll test that in my scenario with different enterprise versions because I'm not running Splunk as a container.

Reply:
@rcabral-splunk How did the test go?

Reply:
I've tested 7.3.3 and 8.0.0 non-containerized versions. This particular change did not impact on anything, although the comments at https://github.com/splunk/splunk-ansible/pull/318#discussion_r357479197 are still a blocker